fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1706,26 +1706,38 @@ public class Config extends ConfigBase {
     /**
      * iceberg metadata cache dir
      */
-    @ConfField
+    @ConfField(mutable = true)
     public static String iceberg_metadata_cache_disk_path = StarRocksFE.STARROCKS_HOME_DIR + "/caches/iceberg";
 
     /**
-     * iceberg metadata cache total size, default 2GB
+     * iceberg metadata memory cache total size, default 512MB
      */
-    @ConfField
-    public static long iceberg_metadata_cache_capacity = 2147483648L;
+    @ConfField(mutable = true)
+    public static long iceberg_metadata_memory_cache_capacity = 536870912L;
 
     /**
-     * iceberg metadata cache max entry size, default 8MB
+     * iceberg metadata memory cache expiration time, default 87400s
      */
-    @ConfField
-    public static long iceberg_metadata_cache_max_entry_size = 8388608L;
+    @ConfField(mutable = true)
+    public static long iceberg_metadata_memory_cache_expiration_seconds = 87400;
```

Comment:
24h? 86400, better to use `24 * 3600`

Reply:
I think some job are periodically, may be it is daily, so we'd better cache a little more, do you think so?

Reply:
if so, is 100 sec enough?